### PR TITLE
feat: show user attributes on settings page

### PR DESCRIPTION
**Pull request by opqdonut**
follow-up to #2403
for #2142

# Definition of Done / Review checklist

## Reviewability
- [x] link to issue
- [x] consider adding screenshots for ease of review

## Documentation
- [x] update changelog if necessary
- [x] components are added to guide page

### CODE REVIEW

**Comment by opqdonut** [comment]
details

**Comment by opqdonut** [comment]
This is kind of the MVP for users seeing their own attributes. I think the "Settings" page could probably be renamed to something else, perhaps "Your account"?

File: src/cljs/rems/user.cljs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            [rems.common.application-util :refer [get-member-name]]
+            [rems.common.util :refer [index-by]]
+            [rems.text :refer [text localized]])
+  (:require-macros [rems.guide-macros :refer [component-info example]]))
+
+(defn username [attributes]
+  (when-let [name (get-member-name attributes)]
+    [info-field (text :t.applicant-info/name) name {:inline? true}]))
+
+(defn attributes [attributes]
```
Comment (Macroz) [inline src/cljs/rems/user.cljs:14]: This could have a docstring. I'm kind of missing the detail about where this is supposed to be used.

**Review by Macroz — APPROVED** [review]
I think after adding this details section we should brush up the notification settings a bit too. I'm not sure it is the most comprehensible one at the moment. At minimum it could be e.g.under heading "Notifications".

Also I don't quite like that this page has its own look, e.g. have a look at this screenshot. Perhaps it would work with two regular collapses with the "Notifications" and "Your user details" headings? Small change but rather more unified look.

three

**Comment by opqdonut** [comment]
So how about calling the page something like "User" or "Profile" and then having "Notification settings" and "User Attributes" as subtitles?

**Comment by Macroz** [comment]
> So how about calling the page something like "User" or "Profile" and then having "Notification settings" and "User Attributes" as subtitles?

I think that's a good idea too.

**Comment by opqdonut** [comment]
profile

**Review by Macroz — APPROVED** [review]
I like it more!

**Pull request merged**